### ISSUE TITLE
Update example of custom template pagination

### ISSUE THREAD

**Issue by rkrayovskyy** [michaelbromley/ng2-pagination#105]
**Angular 2 version**: 2.2.3

**ng2-pagination version**: 1.0.1

**Description of issue**: Since 1.0.0, usage of PaginationControlsDirective has been changed and new way of using custom template has been introduced and IPaginationInstance to configure it is not available anymore. Is it possible to update example of a custom template?

**Comment by rkrayovskyy** [comment]
Apparently, I missed one detail from the documentation:
you should import `PaginationInstance` instead of `IPaginationInstance` as it was in earlier releases. All the rest seems to be working fine, however was not that obvious from the beginning as documentation at github differs from example at demo page.

Closing the issue but may be useful for others.

**Comment by michaelbromley** [comment]
Hi, can you point me to the part where the docs differ from them demo? I couldn't find anything but if it is there, I will fix it :)

**Comment by rkrayovskyy** [comment]
ok, here is what made me confused:

I looked into 'Documentation' at PaginationControlsDirective for building custom template, and then looked into 'Custom Template Example':

- I did not notice no "I" in PaginationInstance instead of IPaginationInstance
- PaginationControlsDirective has 'maxSize' property but custom example has 'itemsPerPage' property
so, I though that example is not updated since 0.x.x version and created an issue here.

Probably it is just because I was migrating and could not catch the diff. May be worth to add a link from basic documentation to custom example so people can connect the dots.

Overall, awesome work, thanks a lot!
